src/lib.rs

```diff
@@ -779,10 +788,40 @@ pub struct Library {
 
 impl Library {
     fn from_pkg_config(name: &str, l: pkg_config::Library) -> Self {
+        let system_roots = if cfg!(target_os = "macos") {
```

Comment:
Would be good to mention `pkg-config-rs` as this code is copied from it.

Reply:
Added a comment that links to the original source